lambdas/service/app.py

```diff
@@ -1057,26 +1057,27 @@ def parameter_hoisting_note(method: str,
                             ) -> str:
     return fd('''
         Any of the query parameters documented below can alternatively be passed
-        as a property of a JSON object in the body of the request. This can be
-        useful in case the value of the `filters` query parameter causes the URL
-        to exceed the maximum length of 8192 characters, resulting in a 413
-        Request Entity Too Large response.
+        as a property of a JSON object in the body of the request. This is
+        referred to as *parameter hoisting* and can be useful in case the value
+        of the `filters` query parameter causes the URL to exceed the maximum
+        length of 8192 characters, resulting in a 413 Request Entity Too Large
+        response.
 
         The request `%s %s?filters={…}`, for example, is equivalent to  `%s %s`
-        with the body `{"filters": "{…}"}` in which any double quotes or
-        backslash characters inside `…` are escaped with another backslash. That
-        escaping is the requisite procedure for embedding one JSON structure
-        inside another.
+        with a `Content-Type` header of `application/json` and the body
+        `{"filters": "{…}"}` in which any double quotes or backslash characters
+        inside `…` are escaped with another backslash. That escaping is the
+        requisite procedure for embedding one JSON structure inside another.
```

Comment:
Correct me if I am wrong but it seems like these two hunks are unrelated documentation improvement that I requested at some point, but that have nothing to do with "Add[ing] support for POST requests to the manifest endpoint", as is the title of the commit the hunks are part of.